src/Test/Utilities/Portable/Compilation/OperationTreeVerifier.cs

```diff
@@ -44,7 +50,8 @@ public static void Verify(Compilation compilation, IOperation operation, string
 
         public static string GetOperationTree(Compilation compilation, IOperation operation, int initialIndent = 0)
         {
-            var walker = new OperationTreeVerifier(compilation, operation, initialIndent);
+            var walker = new OperationTreeVerifier(compilation, operation, initialIndent, 
```

Comment:
Is this run in both Debug and Release modes?

Reply:
> Is this run in both Debug and Release modes?

I believe so.